codespell_lib/data/dictionary.txt

```diff
@@ -11707,13 +11775,20 @@ establishs->establishes
 establising->establishing
 establsihed->established
 estbalishment->establishment
+esthetic->aesthetic
+esthetically->aesthetically
+estheticly->aesthetically
+esthetics->aesthetics
```

Comment:
I think all of these should be moved to `dictionary_en-GB_to_en-US.txt` since `esthetic` is correct in US spelling.

Reply:
They can't be moved. See #2657.